pkg/rtc/config.go

```diff
@@ -112,6 +131,41 @@ func NewWebRTCConfig(conf *config.Config, externalIP string) (*WebRTCConfig, err
 	}
 	s.SetNetworkTypes(networkTypes)
 
+	// publisher configuration
+	publisherConfig := DirectionConfig{
+		RTPHeaderExtension: RTPHeaderExtensionConfig{
+			Audio: []string{
+				sdp.SDESMidURI,
+				sdp.SDESRTPStreamIDURI,
+				sdp.AudioLevelURI,
+			},
+			Video: []string{
+				sdp.SDESMidURI,
+				sdp.SDESRTPStreamIDURI,
+				sdp.TransportCCURI,
+				frameMarking,
+			},
+		},
+	}
+
+	// subscriber configuration
+	subscriberConfig := DirectionConfig{
+		RTCPFeedback: RTCPFeedbackConfig{
+			Video: []webrtc.RTCPFeedback{
```

Comment:
I believe we need the RTCP video feedback for the publisher side as well?

Reply:
Yeah, missed that. Will add it. Got thrown off by that code setting REMB. Will see publisher side to `transport-cc`.

Reply:
Seems to be not needed as Pion is replying with what is in the offer. But, does not hurt adding it. Also, adding both REMB and TWCC as both are supported in Pion.